src/liquid/components/ld-select/ld-select.tsx

```diff
@@ -84,46 +74,59 @@ export class LdSelect implements InnerFocusable {
   /** Used to specify the name of the control. */
   @Prop() name: string
 
-  /**
-   * Used as trigger button label in multiselect mode
-   * and in single select mode if nothing is selected.
-   */
+  /** Used as trigger button label in multiselect mode and in single select mode if nothing is selected. */
   @Prop() placeholder: string
 
   /** Attached as CSS class to the select popper element. */
   @Prop() popperClass?: string
 
+  /** Prevents a state with no options selected after initial selection in single select mode. */
+  @Prop() preventDeselection: boolean
+
+  /** A Boolean attribute indicating that an option with a non-empty string value must be selected. */
+  @Prop() required?: boolean
+
   /**
-   * Prevents a state with no options selected after
-   * initial selection in single select mode.
+   * Currently selected option(s)
+   * @internal
    */
-  @Prop() preventDeselection: boolean
+  @Prop({ mutable: true }) selected: SelectOption[] = []
 
   /** Size of the select trigger button. */
   @Prop() size?: 'sm' | 'lg'
 
-  /**
-   * Stringified tether options object to be merged with the default options.
-   */
+  /** Stringified tether options object to be merged with the default options. */
   @Prop({ mutable: true }) tetherOptions = '{}'
 
-  /**
-   * A Boolean attribute indicating that an option with a non-empty string value must be selected.
-   */
-  @Prop() required?: boolean
-
   @State() ariaDisabled = false
   @State() expanded = false
   @State() hasCustomIcon = false
   @State() hasMore = false
   @State() initialized = false
   @State() internalOptionsHTML: string
   @State() renderHiddenInput = false
-  @State() selected: SelectOption[] = []
   @State() theme: string
   @State() typeAheadQuery: string
   @State() typeAheadTimeout: number
 
+  /**
+   * Emitted with an array of selected values when an alteration to the selection is committed by the user.
+   */
+  @Event() ldchange: EventEmitter<string[]>
```

Comment:
We need to make sure that we stay with one convention for event names. I've been using camelCase for custom events so far, which I find is more readable. Only thing that makes me nervous about camelCase event names so far is [this issue](https://github.com/ionic-team/stencil/issues/2804). There is a reference to a recommendation from the Vue team in [this comment](https://github.com/ionic-team/stencil/issues/2804#issuecomment-763058994) which suggests to use kebab-case for event names. I think kebab-case is better than alllowercase, because it's also safe but more readable. What do you think?

Reply:
Result after call: We will switch to `ld-event-name`, but for existing events that we just prefix, we use e.g. `ld-focuout` maintain the recognizablility high. We'll refactor all custom event names at once in a separate PR.